documents/motions/10.2019.2 - Disciplinary Committee.md

```diff
@@ -25,5 +25,7 @@ The WCA Disciplinary Committee is an Advisory Committee of the WCA.
       3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
       4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
 2. All WCA Staff must fully cooperate with investigations by the WCA Disciplinary Committee.
-3. All Registered Speedcubers should fully cooperate with investigations by the WCA Disciplinary Committee.
+3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary Committee.
+   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee in a timely manner.
+      1. If a Registered Speedcuber decides to stop responding to the WCA Disciplinary Committee, they must inform the WCA Disciplinary Committee or the WCA Board as to the reasons why.
```

Comment:
The phrasing of this sounds odd. If the speedcuber is no longer responding to the WDC, then they would by definition not inform the WDC. Maybe something like:
```suggestion
      1. A Registered Speedcuber must inform the WCA Disiplinary Committee or the WCA Board if they decide to stop responding to the WCA Disciplinary Committee and explain as the the reasons why
```

Reply:
Sure, I think this is a reasonable clarification.

Reply:
`as to the reasons why` seems a bit faux-legalese to me, and it strikes me as more difficult to parse. (Also note that the numbering is wrong in the current PR, but I believe Markdown fixes that automatically.)

What's the main purpose of this? Isn't it a contradiction of `must fully cooperate with investigations` above?
Is the idea that the Registered Speedcuber can decide to stop cooperating, but there will be harsher consequences if they don't communicate that they will do this?

Reply:
(Where's the numbering wrong?)

I don't think this contradicts the point above as I see it more as a way of enforcing that RS will have the duty to at least inform the WDC that they will be no longer replying to emails instead of just letting them go un-replied.

Reply:
I'd prefer to keep it.

Reply:
I am also in favor of keeping it. Lucas, please respond to Alberto's question.

Reply:
I have already included the clarification in a `Staff review` commit that I'll be pushing live later, and I double checked but couldn't find where the numbering is wrong.